moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -1070,11 +874,54 @@ export const ProjectActivityLogCreateDescriptions = {
       return `'${phaseName}' as Project Phase with start date as '${recordData.phase_start}' and end date as '${recordData.phase_end}'`;
     },
   },
+
```

Comment:
I was going to be nit-picky and question the empty lines in the objects, but i ran prettier on the file and it kept the empty lines, so I guess its just personal preference and can totally stay in.

Reply:
Now that you mention it, I'm also surprised the linter didn't kick that back. I'm inclined to go with your instincts here; putting those line feeds in there makes it look like a bunch of independent functions and obfuscates the object definition. 🙏